start-client/src/components/common/error/Warnings.js

```diff
@@ -0,0 +1,90 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import get from 'lodash.get'
+
+import { IconTimes } from '../icons'
+
+class Warnings extends React.Component {
+  render = () => {
+    const warnings = []
+    const keys = Object.keys(this.props.values || {})
+
+    if (keys.length > 0) {
+      for (let key of keys) {
+        const value = get(this.props.values, key)
+        if (key === 'dependencies') {
+          warnings.push(
+            `The following dependencies can not be add: <strong>${value.map(
+              dep => dep.value
+            )}</strong>`
+          )
+        }
+        if (key === 'project') {
+          warnings.push(
+            `<strong>${value.value}</strong> is not a valid project type. The default value has been selected.`
+          )
+        }
+        if (key === 'language') {
+          warnings.push(
+            `<strong>${value.value}</strong> is not a valid language. The default value has been selected.`
```

Comment:
Similarly, could this say "Java has been selected" rather than "The default value has been selected"?